user/languages/haskell.md

```diff
@@ -57,6 +57,18 @@ By default Travis CI uses `cabal` to manage your project's dependencies:
 cabal install --only-dependencies --enable-tests
 ```
 
+### Specifying cabal-install verison
```

Comment:
```suggestion
### Specifying cabal install version
```
@plaindocs

Reply:
`cabal-install` is a proper-noun. https://hackage.haskell.org/package/cabal-install